install_scripts/templates/kubernetes/init.sh

```diff
@@ -117,34 +146,38 @@ EOF
 - $PUBLIC_ADDRESS
 EOF
     fi
-
-
-
 }
 
 initKube() {
     logStep "Verify Kubelet"
     if [ ! -e "/etc/kubernetes/manifests/kube-apiserver.yaml" ]; then
         logStep "Initialize Kubernetes"
+
         initKubeadmConfig
+
         loadIPVSKubeProxyModules
-        set +e
 
+        set +e
```

Comment:
wanna add a comment here why even though this is not new?

Reply:
Don't remember, I'll just remove it.